chukwuemekachm/utility_manager_web
# Prof: Fix ConfirmResetPassword page

#### What does this PR achieve?
- extracts AuthenticationFormLayout to  a separate re-usable components
- uses the extracted component for Authentication page
- re-uses the AuthenticationFormLayout for ConfirmResetPassword page
- ensures that reset password logic still works
- adds some story docs for new and existing components
- ensures that ConfirmResetPassword view is exactly as the mockup.
- ensures that ConfirmResetPassword logic still works

#### Any background context
- Confirm reset password page did not look exactly as is in the mockup

#### Manual Testing Steps
- `npm run dev`
- Begin reset password process from postman by hitting `{{base-url}}/api/auth/reset` with:
```
{
	"email": <your-email>.
	"redirectURL": "http://localhost:8080/reset-password"
}
```
- When you click on the link from your email, you should get the following pages in the screenshots below.
- Input new password, hit submit and verify that new password is what is required for login

#### Screenshots:
<img width="1440" alt="Screenshot 2019-12-30 at 4 54 30 PM" src="https://user-images.githubusercontent.com/38565349/71589407-12f43500-2b25-11ea-8539-07735582d4f6.png">
<img width="1440" alt="Screenshot 2019-12-30 at 4 54 19 PM" src="https://user-images.githubusercontent.com/38565349/71589408-12f43500-2b25-11ea-9b26-1b7560d75e19.png">

#### Relevant Trello Tickets
- [Update Password](https://trello.com/c/69fYaFQV/44-fix-confirmpassword-page)
